dracut.sh

```diff
@@ -2010,10 +2010,13 @@ else
     done
 fi
 
+# shellcheck disable=SC2174
+mkdir -m 0755 -p "${initdir}"/lib "${initdir}"/lib/dracut
+
 if [[ $kernel_only != yes ]]; then
     mkdir -p "${initdir}/etc/cmdline.d"
     # shellcheck disable=SC2174
-    mkdir -m 0755 -p "${initdir}"/lib "${initdir}"/lib/dracut "${initdir}"/lib/dracut/hooks
```

Comment:
Why move it out from here (and not just from within the loop) ? 

> This directory is always needed 

Even if $kernel_only is set to yes  ?

Reply:
> Why move it out from here (and not just from within the loop) ?

It's the same directory that is trying to be created inside the loop, necessary at an earlier point in the code.

> > This directory is always needed
> 
> Even if $kernel_only is set to yes ?

Yes. At least to save the [list of modules included](https://github.com/dracutdevs/dracut/blob/69e9fc84e2920b080c8726a21f9d770beed2b8f6/lsinitrd.sh#L188) and the [build arguments](https://github.com/dracutdevs/dracut/blob/69e9fc84e2920b080c8726a21f9d770beed2b8f6/lsinitrd.sh#L377).